data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/LogstashConfigConverter.java

```diff
@@ -33,7 +33,8 @@ public String convertLogstashConfigurationToPipeline(String logstashConfiguratio
         LogstashParser parser = new LogstashParser(tokens);
         final ParseTree tree = parser.config();
 
-        org.opensearch.dataprepper.logstash.parser.LogstashVisitor visitor = new org.opensearch.dataprepper.logstash.parser.LogstashVisitor();
+        org.opensearch.dataprepper.logstash.parser.ModelConvertingLogstashVisitor visitor =
```

Comment:
Is there a reason you didn't `import org.opensearch.dataprepper.logstash.parser.LogstashVisitor`

Reply:
Prior to renaming `LogstashVisitor`, it had a conflicting class name with an ANTLR-generated class. So it required the fully qualified name. Now that it is renamed, it can be imported.